CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contribution Guidelines
+
+Ensure your pull request adheres to the following guidelines:
+
+- If you submit a project that is similar to an existing project in the list, argue how it's different or better.
+- Suggested packages should be tested and documented.
+- Make an individual pull request for each suggestion.
+- Use the following format: `[package](link) - Description.`
+- Additions should be added to the bottom of the relevant category.
```

Comment:
Does it mean all newly added modules will be at the bottom instead of sorting them alphabetically?

Reply:
Alphabetical ordering is better for me.